tests/v2/validation/resource/terraform/v1charts/Jenkinsfile

```diff
@@ -0,0 +1,149 @@
+// Job Params
+// Provisioning Requirements: RANCHER_API_URL, RANCHER_TOKEN_KEY, CLUSTER_PREFIX, WORKER_COUNT, AWS_AMI, AWS_AMI_USER
+// Target Requirements: RANCHER_API_URL, RANCHER_TOKEN_KEY, CLUSTER_ID, PROJECT_ID
+
+node {
+  def job_name = "${JOB_NAME}"
+  if (job_name.contains('/')) { 
+    job_names = job_name.split('/')
+    job_name = job_names[job_names.size() - 1] 
+  }
+  def testContainer = "${job_name}${env.BUILD_NUMBER}_test"
+  def imageName = "rancher-validation-${job_name}${env.BUILD_NUMBER}"
+  def envFile = ".env"
+
+  def branch = "release/v2.6"
+  if ("${env.branch}" != "null" && "${env.branch}" != "") {
+    branch = "${env.branch}"
+  }
+
+  def rancher_api_url = "${env.RANCHER_API_URL}"
+  def rancher_token_key = "${env.RANCHER_TOKEN_KEY}"
+  def aws_ami = "${env.AWS_AMI_AUTO}"
+  def aws_ami_user = "${env.AWS_AMI_USER_AUTO}"
+  def aws_vpc_id = "${env.AWS_VPC_ID_AUTO}"
+  def aws_zone = "${env.AWS_ZONE_AUTO}"
+  def aws_region = "${env.AWS_REGION_AUTO}"
+  def aws_root_size = "${env.AWS_ROOT_SIZE_AUTO}"
+  def aws_instance_type = "${env.AWS_INSTANCE_TYPE_AUTO}"
+  def aws_subnet = "${env.AWS_SUBNET_AUTO}"
+  def aws_volume_type = "${env.AWS_VOLUME_TYPE_AUTO}"
+  def rancher_k8s_version = "${env.RANCHER_K8S_VERSION}"
+  def cluster_prefix = "${env.CLUSTER_PREFIX}"
+  def worker_count = "${env.WORKER_COUNT}"
+  def tf_version = "${env.TF_VERSION}"
```

Comment:
If possible we should default to a version that meets the terraform module requirement of >= 0.14